### ISSUE TITLE
Loop-shaping controller optimization

### ISSUE THREAD

**Issue by bqpd** [hoburg/gpkit-models#48]
Could in particular be used to optimize / systematize the discovery of pareto frontiers for energy costs of biochemical circuits. Re: [The Energy Costs of Insulators in Biochemical Networks](http://www.sciencedirect.com/science/article/pii/S0006349513001975?np=y)

**Comment by whoburg** [comment]
Isn't it even broader than that specific example? A connection between GP and loop shaping controller design would be an impressive paper.

**Comment by bqpd** [comment]
Certainly! That example's neat because it's actually applying loop-shaping techniques to a nonlinear system, something that wouldn't be possible without SP. :)
